src/elements/content-sidebar/activity-feed/task-new/__tests__/Task-test.js

```diff
@@ -251,4 +256,33 @@ describe('elements/content-sidebar/ActivityFeed/task-new/Task', () => {
 
         expect(wrapper.find('CommentInlineError')).toHaveLength(1);
     });
+
+    test('should call fetchTaskCollaborators on modal open if there is a next_marker', async () => {
+        const taskWithMarker = {
+            ...task,
+            assigned_to: {
+                next_marker: 'foo',
+                entries: [],
+            },
+        };
+
+        const wrapper = mount(
+            <Task
+                {...taskWithMarker}
+                currentUser={currentUser}
+                error={{ title: 'blah', message: 'blah' }}
+                onEdit={jest.fn()}
+                onDelete={jest.fn()}
+            />,
+        );
+        const instance = wrapper.instance();
+        instance.fetchTaskCollaborators = jest
+            .fn()
+            .mockRejectedValueOnce()
+            .mockResolvedValueOnce({});
+
+        await instance.handleEditClick();
+
+        expect(instance.fetchTaskCollaborators).toBeCalled();
```

Comment:
Can we go one step further in the tests and check that the fetched assignees should up where they should?

Reply:
Working on it. I can add some tests as a follow up PR